<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoxFilterDialog</class>
 <widget class="QDialog" name="BoxFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="lyt_depth" stretch="0,0">
     <item>
      <widget class="QLabel" name="lbl_depth_label">
       <property name="text">
        <string>Desired depth:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmb_depth_value"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lyt_k_size" stretch="0,0">
     <item>
      <widget class="QLabel" name="lbl_k_size_label">
       <property name="text">
        <string>K-Size (M, M):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spb_k_size_value">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>0</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lyt_normalized" stretch="1,0">
     <item>
      <widget class="QLabel" name="lbl_normalized_label">
       <property name="text">
        <string>Normalized:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chb_normalized_value">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
